specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1905,12 +1905,34 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
+            "None"
```

Comment:
Change enum with modelAsString=false is likely a breaking change.

Reply:
@weidongxu-microsoft  Can you point us to how to add this extra identity types without breaking?

Reply:
I am checking with other swagger review team members on this. I think there is no way to make it nonbreaking, given the previous definition.

I see it is 2020-01, so question is if you have SDK released or not. If not, you might be safe to modify.

Reply:
I can do one thing. I will release a new version. "2020-02-01". I think that should fix the issue.